tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -85,8 +85,8 @@
 #define DAC_RESOLUTION		12
 #define DAC_CHANNEL_ID		0
 
-#elif defined(CONFIG_BOARD_SAM_E70_XPLAINED) || \
-	defined(CONFIG_BOARD_SAM_V71_XULT)
+#elif defined(CONFIG_BOARD_SAM_E70_XPLAINED) || defined(CONFIG_BOARD_SAM_V71_XULT) ||              \
+	defined(CONFIG_BOARD_SAM4S_XPLAINED)
```

Comment:
Would mind to check if below passes on CI?
The same config is used for all boards and those are select only when `dac` is defined on the `<board>.yaml` file.

```diff
-#elif defined(CONFIG_BOARD_SAM_E70_XPLAINED) || \
-	defined(CONFIG_BOARD_SAM_V71_XULT)
+#if defined(CONFIG_SOC_FAMILY_ATMEL_SAM) && \
+	!defined(CONFIG_SOC_SERIES_SAM4L)
```